test/apps/projection/app.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react';
+import {createRoot} from 'react-dom/client';
+
+import {DeckGL} from '@deck.gl/react';
+import {
+  View,
+  PickingInfo,
+  MapView,
+  MapViewState,
+  _GlobeView as GlobeView,
+  GlobeViewState,
+  OrthographicView,
+  OrthographicViewState
+} from '@deck.gl/core';
+import {ScatterplotLayer} from '@deck.gl/layers';
+import {makePointGrid, makeLineGrid, Point} from './data';
+import {Map, Source, Layer} from '@vis.gl/react-maplibre';
+// Use dev build for source map and debugging
+import maplibregl from 'maplibre-gl/dist/maplibre-gl-dev';
+import {CPULineLayer} from './cpu-line-layer';
+
+const VIEWS: Record<
+  string,
+  {
+    view: View;
+    viewState: any;
+    xRange: [number, number];
+    yRange: [number, number];
+    step: number;
+    baseMap: 'mercator' | 'globe' | false;
+  }
+> = {
+  map: {
+    view: new MapView(),
+    viewState: {
+      longitude: 0,
+      latitude: 0,
+      zoom: 1
+    } satisfies MapViewState,
+    baseMap: 'mercator',
+    xRange: [-180, 180],
+    yRange: [-85, 85],
+    step: 5
+  },
+  'map-high-zoom': {
+    view: new MapView(),
+    viewState: {
+      longitude: 24.87,
+      latitude: 60.175,
+      zoom: 16
+    } satisfies MapViewState,
+    baseMap: 'mercator',
+    xRange: [24.86, 24.88],
+    yRange: [60.17, 60.18],
+    step: 1 / 3000
+  },
+  globe: {
+    view: new GlobeView(),
+    viewState: {
+      longitude: 0,
+      latitude: 0,
+      zoom: 2
+    } satisfies GlobeViewState,
+    baseMap: 'globe',
+    xRange: [-180, 180],
+    yRange: [-85, 85],
+    step: 5
+  },
+  orthographic: {
+    view: new OrthographicView({flipY: false}),
+    viewState: {
+      target: [0, 0, 0],
+      zoom: 0
+    } satisfies OrthographicViewState,
+    baseMap: false,
+    xRange: [-500, 500],
+    yRange: [-400, 400],
+    step: 40
+  },
+  'orthographic-high-zoom': {
+    view: new OrthographicView({flipY: false}),
+    viewState: {
+      target: [20001, 10001, 0],
+      zoom: 16
+    } satisfies OrthographicViewState,
+    baseMap: false,
+    xRange: [20000.99, 20001.01],
+    yRange: [10000.99, 10001.01],
+    step: 1 / 3000
+  }
+} as const;
+
+function getTooltip({object}: PickingInfo): string | null {
+  return object ? JSON.stringify(object) : null;
+}
+
+function App() {
+  const [viewMode, setViewMode] = React.useState<keyof typeof VIEWS>('map');
+
+  const opts = VIEWS[viewMode];
+
+  const pointData = React.useMemo(() => makePointGrid(opts), [viewMode]);
```

Comment:
Would be nice to include a finer grid around zoom 12 to see how much we deviate from the adaptive projection in maplibre